resources/js/oauth2/consent.js

```diff
@@ -0,0 +1,163 @@
+            <Card className={styles.consent_container} variant="outlined">
+                <CardHeader title='This app would like to:'/>
+                <Divider/>
+                <CardContent>
+                    <form
+                        ref={formEl}
+                        method="post"
+                        action={formAction}
+                        autoComplete="off"
+                        target="_self">
+                        <Grid
+                            container
+                            direction="column"
+                            spacing={2}
+                            justifyContent="center">
+                            <Grid item>
+                                <div className={styles.scopes_list_container}>
+                                    {
+                                        requestedScopes.map((s, ix) => {
+                                            return <div className={styles.scope_item} key={ix}
+                                                        aria-hidden="true">{s}&nbsp;
+                                                <HtmlTooltip arrow title={<Typography color="inherit">{s}</Typography>}>
+                                                    <InfoOutlinedIcon fontSize="small"/>
+                                                </HtmlTooltip>
+                                            </div>;
+                                        })
+                                    }
+                                </div>
+                            </Grid>
+                            <Grid item className={styles.disclaimer}>
+                                <Typography variant="body2" dangerouslySetInnerHTML={{__html: disclaimer}}>
+                                </Typography>
+                            </Grid>
+                            <Grid
+                                item
+                                container
+                                direction="row"
+                                spacing={2}
+                                justifyContent="center"
+                                alignItems="center">
+                                <Grid item container xs={6} justifyContent="center">
+                                    <Button
+                                        id="cancel-authorization"
+                                        variant="outlined"
+                                        size="large"
+                                        className={styles.button}
+                                        disableElevation
+                                        type="button"
+                                        onClick={handleCancel}
+                                    >
+                                        Cancel
+                                    </Button>
+                                </Grid>
+                                <Grid item container xs={6} justifyContent="center">
+                                    <Button
```

Comment:
we need to disable the button after click to prevent double clicking

Reply:
Done